core/base/abstractTriangulation/AbstractTriangulation.h

```diff
@@ -2548,6 +2548,14 @@ namespace ttk {
       return this->vertGid_;
     }
 
+    inline const ttk::LongSimplexId *getEdgesGlobalIds() const {
+      return this->edgeLidToGid_.data();
+    }
+
+    inline const ttk::LongSimplexId *getTrianglesGlobalIds() const {
+      return this->triangleLidToGid_.data();
+    }
+
```

Comment:
This seems to be incompatible with what I'm working on in the `ImplicitTriangulation`, meaning the on-the-fly computation of the edges and triangles global (resp. local) identifiers from their local (resp. global) identifiers. My changes would make these vectors useless (except for the `ExplicitTriangulation`).

Reply:
it sounds from this discussion that #866 should be merged prior to this PR.
do you both agree?

Reply:
I looked at #866, and it appears that this PR only implements the changes for cells, and not all simplices. Before #866, `exchangeGhostCells` could be used with any type of simplex or cell. Indeed, it would be good to merge it before this one, however the case of intermediary simplices (edges and triangles) would still be unchanged, therefore I would either need to change exchangeGhostCells again or keep things as they are, with a RankArray vector, for those simplices.